src/test/perl/rbe_value_format.t

```diff
@@ -140,35 +140,64 @@ Readonly my @TEST_ARRAY => ('confFile', 'logFile', 'unused', 'logKeep', 'logFile
+                                                  LINE_FORMAT_KW_VAL,
                                                   LINE_VALUE_ARRAY,
                                                   LINE_VALUE_OPT_SORTED,
                                                  );
 is($formatted_value, $FORMATTED_ARRAY_SORTED, "Array values (sorted) correctly formatted");
 $formatted_value = $rbe_fh->_formatAttributeValue(\@TEST_ARRAY,
-                                                  LINE_FORMAT_KEY_VAL,
+                                                  LINE_FORMAT_KW_VAL,
                                                   LINE_VALUE_ARRAY,
                                                   LINE_VALUE_OPT_UNIQUE,
                                                  );
 is($formatted_value, $FORMATTED_ARRAY_UNIQUE, "Array values (unique) correctly formatted");
+$formatted_value = $rbe_fh->_formatAttributeValue(\@TEST_ARRAY,
+                                                  LINE_FORMAT_KW_VAL,
+                                                  LINE_VALUE_ARRAY,
+                                                  LINE_VALUE_OPT_SINGLE,
+                                                 );
+like($formatted_value, qr/$FORMATTED_ARRAY_SINGLE/, "Array values (single) correctly formatted");
 
 
 # LINE_VALUE_HASH_KEYS
 $formatted_value = $rbe_fh->_formatAttributeValue(\%INSTANCE_PARAMS,
-                                                  LINE_FORMAT_KEY_VAL,
+                                                  LINE_FORMAT_KW_VAL,
                                                   LINE_VALUE_HASH_KEYS,
                                                   0,
                                                  );
 is($formatted_value, $FORMATTED_ARRAY_UNIQUE, "Hash keys correctly formatted");
 
 
+# LINE_VALUE_HASH
+Readonly my %STRING_HASH => ('abc' => 'cde',
+                             '-crl' => 3,
+                            );
+Readonly my $FORMATTED_STRING_HASH => '-crl 3 abc cde';
+# When LINE_VALUE_OPT_SINGLE is set, formatAttributeValue() does nothihng and returns the hash
+Readonly my $FORMATTED_STRING_HASH_SINGLE => 'HASH\(0x[a-f\d]+\)';
+$formatted_value = $rbe_fh->_formatAttributeValue(\%STRING_HASH,
+                                                  LINE_FORMAT_KW_VAL,
+                                                  LINE_VALUE_HASH,
+                                                  0,
+                                                 );
+is($formatted_value, $FORMATTED_STRING_HASH, "String hash correctly formatted");
+$formatted_value = $rbe_fh->_formatAttributeValue(\%STRING_HASH,
+                                                  LINE_FORMAT_KW_VAL,
+                                                  LINE_VALUE_HASH,
+                                                  LINE_VALUE_OPT_SINGLE,
+                                                 );
+like($formatted_value, qr/$FORMATTED_STRING_HASH_SINGLE/, "String hash (1 key/val per line) correctly formatted");
```

Comment:
same remark about comment above the pattern and the pattern itself being a stringified hashref